tango_simlib/sim_xmi_parser.py

```diff
@@ -256,6 +318,8 @@ def extract_command_description_data(self, description_data):
         output_parameter = description_data.find('argout')
         command_data['argoutDescription'] = output_parameter.attrib['description']
         command_data['argoutType'] = self._get_arg_type(output_parameter)
+        command_data['inherited'] = (
+                description_data.find('status').attrib['inherited'])
         return command_data
```

Comment:
Now that we want to use the XMI parser to "re-write" the XMI files I am concerned that we only extract certain fields for each category - in this case of command_data, we have only argoutDescription, argoutType and inherited. This was fine for tango-simlib simulators as you know what you used to implement the simulators, but there may be (and probably are) fields that we are not aware of and should not loose when we re-write the XMI file. It may be necessary to store all fields, or the complete item, and make sure we write out the complete item and not just the "known" fields. 
Is it possible to loop through all attribs and build the dict from there. If you need specific validation to ensure certain fields are included for tango-simlib simulators, then that has to be done after building the complete dict.